backend/modules/evaluation/infra/rpc/data/dataset.go

```diff
@@ -301,6 +301,11 @@ func (a *DatasetRPCAdapter) BatchCreateDatasetItems(ctx context.Context, param *
 	return resp.GetAddedItems(), convert2EvaluationSetErrorGroups(ctx, resp.GetErrors()), nil, nil
 }
 
+func (a *DatasetRPCAdapter) BatchUpdateDatasetItems(ctx context.Context, param *rpc.BatchUpdateDatasetItemsParam) (errorGroup []*entity.ItemErrorGroup, itemOutputs []*entity.DatasetItemOutput, err error) {
+	// TODO implement me
+	panic("implement me")
```

Comment:
[🚫Must-Fix] Unimplemented method panics at runtime
`BatchUpdateDatasetItems` currently calls `panic("implement me")`. Any update request through `BatchUpdateEvaluationSetItemsOApi` will crash the service.

Concrete fix: implement the RPC call or return a proper error without panicking to preserve service stability.

Example stub (avoid panic):
```suggestion
func (a *DatasetRPCAdapter) BatchUpdateDatasetItems(ctx context.Context, param *rpc.BatchUpdateDatasetItemsParam) (errorGroup []*entity.ItemErrorGroup, itemOutputs []*entity.DatasetItemOutput, err error) {
    if param == nil {
        return nil, nil, errorx.NewByCode(errno.CommonInvalidParamCode)
    }
    // TODO: call underlying dataset service and map response
    return nil, nil, errorx.NewByCode(errno.CommonNotImplementedCode, errorx.WithExtraMsg("BatchUpdateDatasetItems not implemented"))
}
```
References: Go Code Review Comments (do not use panic in production paths), ByteDance Go 代码规范（错误处理与稳定性）。